card/lib/generators/card/USAGE

```diff
@@ -0,0 +1,21 @@
+Description
+
+Examples
+  wagn generate card new cards pre_fix
```

Comment:
Do I understand that this is supposed to be the main mechanism for creating a new engine deck?

I would think the process for setting up a new deck would be something more like:
1. add 'decko-rails' to Gemfile of rails app
2. run bundle install
3. run `decko-rails new wiki`.  

The last one would run a rake task to set up db tables, generate the necessary stuff, and probably output some instructions (add X to your config/routes.rb)

The big question is: what stuff do we need that generator to create?  Here's a starting place: http://guides.rubyonrails.org/engines.html#generating-an-engine.

Those generators make lots of things we know we don't want: app directories and helpers and view and so forth.  But I do suspect we want a new engine class at a minimum and perhaps more.

Step 1 is getting all that to work.  Step 2 is automating its generation :)